packages/languages/src/index.ts

```diff
@@ -25,6 +26,7 @@ type SubLanguage = BaseLanguage & { parentLangSlug: string };
 
 export type Language = BaseLanguage | SubLanguage;
 
-const languages: Language[] = values( data );
-
+export const languagesBySlug = data;
+export const languageSlugs: LanguageSlug[] = keys( data );
```

Comment:
You could always use `Object.keys( data )` instead of lodash. Doesn't make a big difference in the grand scheme of things, but it is one less dependency on a non-standard library.

Reply:
Yea, was just sticking with lodash because the values call did the same, probably should be Object.values / Object.keys. Is this a safe change or does lodash do something extra?

Reply:
The Lodash versions accept a `null` or `undefined` argument and return an empty array for them. The standard `Object.*` versions will throw a `TypeError: Cannot convert undefined or null to object`.

Without Lodash, you need to check if the inputs are always valid objects/arrays and add some guards otherwise.

Reply:
So definitely safe now we're not downloading the file in the build. Will swap to those! thanks!